Algorithms/GRAPHS/Dijkstra/DIJKSTRA.CPP

```diff
@@ -0,0 +1,61 @@
+#include<iostream>
```

Comment:
Dijkstra is also already present in the Single Source Shortest distance folder in graphs.